README.md

```diff
@@ -9,7 +9,21 @@ We'd love to see you contributing to our projects, please just follow a few conv
 - No trailing whitespace. Blank lines should not have any spaces.
 - Do not indent after private/protected.
 - Use Ruby >= 1.9 syntax for hashes. Prefer `{ a: :b }` over `{ :a => :b }`.
-- Prefer `&&`/`||` over and/or.
+- Prefer `&&`/`||` over `and`/`or`.
+- Use `and`/`or` only for the early returns:
```

Comment:
I think it's good to add this 13th line, but we all know what early return is, no need for an explanation here IMO.

Reply:
Not sure if it's obvious enough with `or raise Error`. And if it's already there, I don't think removing it will make it better.

Reply:
It's making our guide more lengthy, so less concise. But I see you've already argued a lot about this with Mandaryn to make better examples 😂 I just want to see this merged.

Reply:
So let's merge it :P.